sdk-lib/src/main/java/cash/z/ecc/android/sdk/db/entity/Transactions.kt

```diff
@@ -81,8 +82,8 @@ data class PendingTransactionEntity(
     override val value: Long = -1,
     override val memo: ByteArray? = byteArrayOf(),
     override val accountIndex: Int,
-    override val minedHeight: Int = -1,
-    override val expiryHeight: Int = -1,
+    override val minedHeight: Long = -1,
```

Comment:
TODO: I missed refactoring these.

Reply:
Actually never mind, these are database objects so they have to map to SQLite columns